Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/segmentio/analytics-swift.git",
         "state": {
           "branch": null,
-          "revision": "76316a5009502de6e7c2788c295e9afc68793cd3",
-          "version": "1.1.2"
+          "revision": "26f23a2ae9b8dd47aeee1bdaa1e203f3bed6044e",
+          "version": "1.4.0"
```

Comment:
You need to be on 1.4.1.  That's the version with the changes you want.

Reply:
ok let me update that things also